.teamcity/src/main/kotlin/configurations/PerformanceTest.kt

```diff
@@ -68,20 +69,22 @@ class PerformanceTest(
                 killGradleProcessesStep(os)
                 substDirOnWindows(os)
 
-                gradleWrapper {
-                    name = "GRADLE_RUNNER"
-                    tasks = ""
-                    workingDir = os.perfTestWorkingDir
-                    gradleParams = (
-                        performanceTestCommandLine(
-                            "clean ${performanceTestTaskNames.joinToString(" ") { "$it --channel %performance.channel% ${type.extraParameters}" }}",
-                            "%performance.baselines%",
-                            extraParameters,
-                            os
-                        ) +
-                            buildToolGradleParameters() +
-                            buildScanTag("PerformanceTest")
-                        ).joinToString(separator = " ")
+                repeat(if (performanceTestBuildSpec.type == PerformanceTestType.flakinessDetection) 2 else 1) {
```

Comment:
Please capture the index here as a parameter. Below it is not clear what `it` references to and I think it will reference to the one from `gradleWrapper`.

Reply:
`gradleWrapper` is actuall using `this`, but it's always better to use an explicit parameter. Fixed.